internal/kafka/command_cluster.go

```diff
@@ -14,18 +14,27 @@ const (
 	multiZone        = "multi-zone"
 	lowAvailability  = "SINGLE_ZONE"
 	highAvailability = "MULTI_ZONE"
+	low              = "LOW"
+	high             = "HIGH"
 )
 
 var availabilitiesToHuman = map[string]string{
 	lowAvailability:  singleZone,
 	highAvailability: multiZone,
+	low:              singleZone,
+	high:             multiZone,
```

Comment:
Since this is meant to be an under-the-hood change for now, these lines are to map "LOW" and "HIGH" to the current "single-zone" and "multi-zone" for output